src/sentry/static/sentry/app/components/footer.tsx

```diff
@@ -56,6 +57,17 @@ const FooterLink = styled(ExternalLink)`
   }
 `;
 
+export const FooterIcon = styled(props => (
+  <a href="/" tabIndex={-1} {...props}>
```

Comment:
shouldn't the link be placed outside of this component? Otherwise, I would rename the component.

Reply:
I thought I was going to need this component elsewhere, but didn't end up using it. I'll rename the component.